partcad-cli/src/partcad_cli/click/commands/export.py

```
@@ -0,0 +1,125 @@
+    "-t",
+    "--format",
+    help="The type of file to export",
+    type=click.Choice(
+        [
+            "step",
+            "stl",
+            "3mf",
+            "threejs",
+            "obj",
+            "gltf",
+        ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)